src/Microsoft.VisualStudio.Web.CodeGenerators.Mvc/Templates/RazorPageGenerator/DeletePageModel.cshtml

```diff
@@ -9,6 +9,12 @@ using Microsoft.AspNetCore.Mvc;
 using Microsoft.AspNetCore.Mvc.RazorPages;
 using Microsoft.EntityFrameworkCore;
 @{
+    var entitySetName = Model.ModelMetadata.EntitySetName;
+    var entitySetVar = Model.EntitySetVariable ??
+        (String.IsNullOrEmpty(entitySetName)
```

Comment:
What is the expected behavior if `entitySetName` *is* null-or-empty? It'll just use an empty string as the `entitySetVar`? And then where is `entitySetVar` even used?

Reply:
`entitySetVar` is not needed. Copied it unnecessarily. 
Will remove. 

For RazorPages with model, since we always need EF based metadata, `Model.ModelMetadata.EntitySetName` is ensured to be not null (or empty)